mixed_requests/onegin

```diff
@@ -0,0 +1,881 @@
+Охотники до похорон.
+Покойника похоронили.
+Попы и гости ели, пили
+И после важно разошлись,
+Как будто делом занялись.
+Вот наш Онегин — сельский житель,
+Заводов, вод, лесов, земель
+Хозяин полный, а досель
+Порядка враг и расточитель,
+И очень рад, что прежний путь
+Переменил на что-нибудь.
+LIV
+
+Два дня ему казались новы
+Уединенные поля,
+Прохлада сумрачной дубровы,
+Журчанье тихого ручья;
+На третий роща, холм и поле
+Его не занимали боле;
+Потом уж наводили сон;
+Потом увидел ясно он,
+Что и в деревне скука та же,
+Хоть нет ни улиц, ни дворцов,
+Ни карт, ни балов, ни стихов.
+Хандра ждала его на страже,
+И бегала за ним она,
+Как тень иль верная жена.
+LV
+
+Я был рожден для жизни мирной,
+Для деревенской тишины;
+В глуши звучнее голос лирный,
+Живее творческие сны.
+Досугам посвятясь невинным,
+Брожу над озером пустынным,
+И far niente мой закон.
+Я каждым утром пробужден
+Для сладкой неги и свободы:
+Читаю мало, долго сплю,
+Летучей славы не ловлю.
+Не так ли я в былые годы
+Провел в бездействии, в тени
+Мои счастливейшие дни?
+LVI
+
+Цветы, любовь, деревня, праздность,
+Поля! я предан вам душой.
+Всегда я рад заметить разность
+Между Онегиным и мной,
+Чтобы насмешливый читатель
+Или какой-нибудь издатель
+Замысловатой клеветы,
+Сличая здесь мои черты,
+Не повторял потом безбожно,
+Что намарал я свой портрет,
+Как Байрон, гордости поэт,
+Как будто нам уж невозможно
+Писать поэмы о другом,
+Как только о себе самом.
+LVII
+
+Замечу кстати: все поэты —
+Любви мечтательной друзья.
+Бывало, милые предметы
+Мне снились, и душа моя
+Их образ тайный сохранила;
+Их после муза оживила:
+Так я, беспечен, воспевал
+И деву гор, мой идеал,
+И пленниц берегов Салгира.
+Теперь от вас, мои друзья,
+Вопрос нередко слышу я:
+«О ком твоя вздыхает лира?
+Кому, в толпе ревнивых дев,
+Ты посвятил ее напев?
+LVIII
+
+Чей взор, волнуя вдохновенье,
+Умильной лаской наградил
+Твое задумчивое пенье?
+Кого твой стих боготворил?»
+И, други, никого, ей-богу!
+Любви безумную тревогу
+Я безотрадно испытал.
+Блажен, кто с нею сочетал
+Горячку рифм: он тем удвоил
+Поэзии священный бред,
+Петрарке шествуя вослед,
+А муки сердца успокоил,
+Поймал и славу между тем;
+Но я, любя, был глуп и нем.
+LIX
+
+Прошла любовь, явилась муза,
+И прояснился темный ум.
+Свободен, вновь ищу союза
+Волшебных звуков, чувств и дум;
+Пишу, и сердце не тоскует,
+Перо, забывшись, не рисует,
+Близ неоконченных стихов,
+Ни женских ножек, ни голов;
+Погасший пепел уж не вспыхнет,
+Я всё грущу; но слез уж нет,
+И скоро, скоро бури след
+В душе моей совсем утихнет:
+Тогда-то я начну писать
+Поэму песен в двадцать пять.
+LX
+
+Я думал уж о форме плана
+И как героя назову;
+Покамест моего романа
+Я кончил первую главу;
+Пересмотрел все это строго:
+Противоречий очень много,
+Но их исправить не хочу.
+Цензуре долг свой заплачу
+И журналистам на съеденье
+Плоды трудов моих отдам:
+Иди же к невским берегам,
+Новорожденное творенье,
+И заслужи мне славы дань:
+Кривые толки, шум и брань!
```

Comment:
Please use English only in any commits. I respect Pushking, but the file is used as just a bulk of data, so it's better just to generate it automatically, e.g. you can use /dev/urandom .

Reply:
Replaced with english text